.travis.yml

```diff
@@ -1,26 +1,21 @@
 language: python
 cache: pip
-python:
-  - 2.7
-  - 3.6
-install:
-  - pip install flake8
-before_script:
-  # stop the build if there are Python syntax errors, PEP8 violations, undefined names
-  - flake8 . --count --select=E,F821,F822,F823 --max-line-length=127 --show-source --statistics
-  # exit-zero treats all errors as warnings.  GitHub editor is 127 chars wide
-  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-script:
-  - true  # add other tests here
-
-matrix:
+
+jobs:
   include:
-    - language: node_js
-      python:
-      node_js:
-        - 8
+    - stage: test
+      name: "Unit Tests 2.7"
```

Comment:
once @caa06d9c PR #363  on harvesters land we could get rid of this as everything will be in Python 3.7 (I hope)

for the time being is ok as this PR will land before than @caa06d9c one ... I think

Reply:
ok